APIs/schemas/token_schema.json

```diff
@@ -51,20 +51,21 @@
     "^x-nmos-[a-z]+$": {
       "description": "An object containing the access permissions of the user for the NMOS API identified by this attribute's name",
       "type": "object",
-      "additionalProperties": false,
       "minProperties": 1,
       "properties": {
         "read": {
           "type": "array",
           "items": {
-            "type": "string"
+            "type": "string",
+            "pattern": "^.+$"
```

Comment:
Not sure whether to prefer `"minLength": ` in these places?

(I see we use `"pattern": "^.+$"` in other specs, e.g. IS-04 Node `interfaces.chassis_id`.)